### PR TITLE
Make @phisco a Crossplane maintainer

### PR DESCRIPTION
**Pull request by negz**
### Description of your changes

Philippe has lead an implemented an impactful design for us in the form of Composition Validation. He's also proven to be pragmatic and easy to collaborate with, and willing to drive laborious but high-value processes like introducing Renovate and liaising with the security audit team.

I have:

- [x] Read and followed Crossplane's [contribution process].
- [ ] ~Added or updated unit **and** E2E tests for my change.~
- [ ] ~Run `make reviewable` to ensure this PR is ready for review.~
- [ ] ~Added `backport release-x.y` labels to auto-backport this PR if necessary.~

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

**Comment by negz** [comment]
@jbw976 I've already spoken to the other maintainers about this and they're on board. This PR will need steering committee approval just because it touches OWNERS.md.

**Review by jbw976 — APPROVED** [review]
very deserved @phisco , thanks for the effort you've put into the project so far, looking forward to more of it 🤓

**Pull request merged**
